src/field_theory/is_alg_closed/basic.lean

```diff
@@ -170,6 +171,10 @@ theorem is_alg_closure_iff (K : Type v) [field K] [algebra k K] :
   is_alg_closure k K ↔ is_alg_closed K ∧ algebra.is_algebraic k K :=
 ⟨λ h, ⟨h.1, h.2⟩, λ h, ⟨h.1, h.2⟩⟩
 
+theorem is_alg_closure.normal {R K : Type*} [field R] [field K] [algebra R K]
+  (h : is_alg_closure R K) : normal R K :=
```

Comment:
Is alg closure is a typeclass. Why is it being supplied as a regular argument here instead of via an instance?

Reply:
Good point. I'll fix that.

Reply:
Also, this can be an instance.

Reply:
Ok!